src/main/java/com/linecorp/armeria/server/metrics/Status.java

```diff
@@ -0,0 +1,25 @@
+package com.linecorp.armeria.server.metrics;
+
+import com.linecorp.armeria.common.SessionProtocol;
+
+/**
+ * Server response status for {@link ServiceMetricConsumer}
+ */
+final class Status {
+    private final SessionProtocol sessionProtocol;
+    private final int code;
+
+    private Status(SessionProtocol sessionProtocol, int code) {
+        this.sessionProtocol = sessionProtocol;
+        this.code = code;
+    }
+
+    public static Status of(SessionProtocol protocol, int code) {
+        return new Status(protocol, code);
```

Comment:
If I implement this using `ConcurrentMap`, performance would be negative.
I checked whether I can pre-populate this, but `HttpResponseStatus` does not provide iterables.

how about removing `Status` and let consumer to accept `SessionProtocol, int, ...` instead of `Status, ...`?